packages/dashboard-frontend/src/store/Plugins/chePlugins/helpers.ts

```diff
@@ -13,6 +13,16 @@
 import devfileApi from '@/services/devfileApi';
 import { che } from '@/services/models';
 
+// unsupported by default editors for s390x architecture
+const S390X_UNSUPPORTED_BY_DEFAULT = [
```

Comment:
Maybe it would be better to include the supported architectures in the editors’ metadata https://github.com/eclipse-che/che-operator/blob/main/editors-definitions/che-webstorm-server-latest.yaml#L20 and then filter the editors based on that?
I don’t think dashboard should be responsible for collecting information about which architectures each editor supports.

@ibuziuk WDYT?

Reply:
would be great to keep all changes for this feature in a single PR - it helps with review and tracking. Let's wait when https://github.com/eclipse-che/che-operator/pull/2010 will be merged